src/Header.css

```diff
@@ -22,3 +22,9 @@ h1{
     text-decoration: none;
     color: #ffcb05;
 }
+   
+.links a:active,
+.links a:hover {
```

Comment:
Ahh - SUPER close with this one! There are actually some styling rules in place for styling `<a>` in CSS. Specifically it relates to the order in which they are activated. 

- `a:hover` must come AFTER `a:link` and `a:visited` (two pseudo classes you may never use lol)
- `a:active` must come AFTER `a:hover`